fluent-package/msi/source.wxs

```diff
@@ -159,6 +160,46 @@
     <WixVariable Id="WixUIExclamationIco" Value="assets\icon.ico" />
     <WixVariable Id="WixUIInfoIco" Value="assets\icon.ico" />
 
+    <!-- Save old some registry values -->
+    <Property Id="REGISTRY_START_OLD">
+      <RegistrySearch Id="RegistrySearchStart" Root="HKLM" Key="System\CurrentControlSet\Services\fluentdwinsvc" Name="Start" Type="raw" />
+    </Property>
+    <Property Id="REGISTRY_DELAYEDAUTOSTART_OLD">
+      <RegistrySearch Id="RegistrySearchDelayedAutostart" Root="HKLM" Key="System\CurrentControlSet\Services\fluentdwinsvc" Name="DelayedAutostart" Type="raw" />
+    </Property>
+    <Property Id="REGISTRY_FLUENTDOPT_OLD">
+      <RegistrySearch Id="RegistrySearchFluentdopt" Root="HKLM" Key="System\CurrentControlSet\Services\fluentdwinsvc" Name="fluentdopt" Type="raw" />
+    </Property>
+
+    <!-- Restore some registry values -->
+    <ComponentGroup Id="KeepSomeRegistryValues" Directory="FLUENTPROJECTLOCATION">
+      <!--
+        CAUTION: We cannot use `integer` type for values being read by RegistrySearch. Use `string` instead.
+        RegistrySearch reads a raw value, and it can have a prefix according to the data type.
+        For example, integer (DWORD) value has a `#` prefix.
+        It appears that RegistryValue can use `string` type to handle such raw values.
+        If using `integer` type, it appears that it is forced to be recognized as `REG_SZ` type after all, and it breaks the service.
+       -->
```

Comment:
Hmm...

Reply:
```
CAUTION: We cannot use `integer` type for values being read by RegistrySearch. Use `string` instead.
DETAIL: RegistrySearch reads a raw value and the value could have a prefix according to the data type.
For example, integer (DWORD) value has a `#` prefix.
If using `integer` type, it appears that it is forced to be recognized as `REG_SZ` type after all, and it make hard to handle from wixtoolset (.wxs).
(It should be converted as Value=N without '#' prefix in this case)
Thus, it is better to use `string` type to handle such raw values in this context because of no need to care about '#' prefix.
```

Reply:
Thanks!
I tried to put them together!
a35a13fa52fe6976cfd2ea69d7df595e6df700fb